core/test/unit/services/url/utils_spec.js

```diff
@@ -522,6 +522,22 @@ describe('Url', function () {
 
             urlService.utils.urlFor('api', {cors: true}, true).should.eql('https://my-ghost-blog.com/ghost/api/v0.1/');
         });
+
+        it('api: with active version, blog url is https: should return active client api path', function () {
+            configUtils.set({
+                url: 'https://my-ghost-blog.com'
+            });
+
+            urlService.utils.urlFor('api', {cors: true, version: "active"}, true).should.eql('https://my-ghost-blog.com/ghost/api/v2/client/');
+        });
+
+        it('api: with active version and admin true, blog url is https: should return active admin api path', function () {
+            configUtils.set({
+                url: 'https://my-ghost-blog.com'
+            });
+
+            urlService.utils.urlFor('api', {cors: true, version: "active", admin: true}, true).should.eql('https://my-ghost-blog.com/ghost/api/v2/admin/');
+        });
```

Comment:
Didn't add any explicit test to check for URL returned for `version: "stable"` as it gets tested with other tests implicitly right now. Can be updated with tests upgrade task

Reply:
Maybe add one example with `stable:true`?

Reply:
Sure :)